src/main/java/gov/cabinetoffice/gap/adminbackend/services/ApplicationFormService.java

```diff
@@ -202,11 +225,17 @@ private void validateMaxWordsValidationField(final ApplicationFormQuestionDTO qu
 
     public String addQuestionToApplicationForm(Integer applicationId, String sectionId,
             ApplicationFormQuestionDTO question, HttpSession session) {
+        AdminSession adminSession = HelperUtils.getAdminSessionForAuthenticatedUser();
 
         String questionId = UUID.randomUUID().toString();
 
         this.applicationFormRepository.findById(applicationId).ifPresentOrElse(applicationForm -> {
 
+            if (!adminSession.getGrantAdminId().equals(applicationForm.getCreatedBy())) {
```

Comment:
Can this be removed please now the ownership check change has went in

Reply:
Good spot mate. Removed and fixed the test associated with it